notebook/services/contents/handlers.py

```diff
@@ -146,7 +143,7 @@ def _copy(self, copy_from, copy_to=None):
     @gen.coroutine
     def _upload(self, model, path):
         """Handle upload of a new file to path"""
-        self.log.info(u"Uploading file to %s", path)
+        self.log.info("Uploading file to %s", path)
```

Comment:
f-string? (others below as well)

Reply:
logging call without f-string is a bit more native (lazy logging formatting) and pylint will complain (BTW personally I prefer f-strings)